tools/autograd/derivatives.yaml

```diff
@@ -409,9 +432,11 @@
 - name: div.Tensor(Tensor self, Tensor other) -> Tensor
   self: div_tensor_self_backward(grad, other, self.scalar_type())
   other: div_tensor_other_backward(grad, self, other)
+  result: "self_fw_grad.defined() ? (other_fw_grad.defined() ? self_fw_grad / other.conj() - other_fw_grad * (self / (other * other)).conj() : self_fw_grad / other.conj()) : (other_fw_grad.defined() ? -1 * other_fw_grad * (self / (other * other)).conj() : Tensor())"
```

Comment:
Are you sure about the conjs here?

Reply:
In forward AD, I think all of the conjugations are wrong.

Conjugation in backward AD arises because we are doing a transposition (hermitian) of the Jacobian matrix. In forward AD there is no transposition; therefore, there is no conj.

Reply:
conj problem is also in other formulas

Reply:
In VC, we established that the conj IS necessary per tests (just goes to show how much I understand complex AD, my intuition is all wrong!!)

Reply:
No all the complex discussion is moved out.
It will be handled in a separate PR.